### DISCUSSION TITLE
getting a sqlite3.databaseerror: database disk image is malformed error

### DISCUSSION THREAD

**Discussion by GerhardMcG** [Chia-Network/chia-blockchain#6750] [Support]
Good Day,

I am getting a sqlite3.databaseerror: database disk image is malformed error in chia app and log. This is stopping me from syncing or plotting or farming. Please I need assistance in resolving this matter as soon possible.

I would greatly appreciate any support regarding this matter.

Regards
Gerhard

**Comment by loppefaaret** [comment]
sounds like you might have been unlucky enough to encounter a malformed blockchain database, if the issue persist, the solution is to remove the database of the blockchain, and resync from 0:
BLOCKCHAIN DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
del ~\.chia\mainnet\db\block*
```
Then try opening up the Chia app again

**Reply by GerhardMcG** [reply]
Hi,
Thank you very much for your response regarding the matter. Another source informed me to delete the mainet\db folder and that the also lets you start from 0. How do you open a Powershell window just in case this does not solve it. It has started syncing again.

For very much interest sake, what is a malformed blockchain database?

Regards
Gerhard

**Reply by loppefaaret** [reply]
yeah, just after posting this, i recalled your name from keybase #support
windows comes with "PowerShell" installed, and you can search for it in the startmenu.
a malformed blockchain database, is simply put that the data in the database that holds the blockchain, is malformed, or corrupted

**Comment by BrandtH22** [comment]
@GerhardMcG , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
